### PR TITLE
CCM Release notes

### PR DESCRIPTION
**Pull request by charanya-j**
Release notes for CCM.

Please ignore other cosmetic fixes.

### CODE REVIEW

File: release-notes/cloud-cost-management.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Cloud Cost Management
 tags: [NextGen, "cloud cost management"]
-date: 2023-03-29T10:00
+date: 2023-04-11T10:00
```
[the remark this answers is not in the document]
Comment (charanya-j) [reply release-notes/cloud-cost-management.md:4]: it's today's date, right?

File: release-notes/whats-new.md [hunk 2]
```diff
@@ -23,6 +23,13 @@ Additionally, the release notes below are only for NextGen SaaS. FirstGen SaaS r
 
 * Filters applied to the monitored services list on the **Monitored Services** page will get reset when you switch to a different project. (SRM-14383)
 
+## April 05, 2023, version 79001
```
Comment (roshnisarangadharan) [inline release-notes/whats-new.md:26]: please check date

File: release-notes/cloud-cost-management.md [hunk 3]
```diff
@@ -13,6 +13,30 @@ Harness deploys updates progressively to different Harness SaaS clusters. You ca
 Additionally, the release notes below are only for NextGen SaaS. FirstGen SaaS release notes are available [here](/docs/first-gen/firstgen-release-notes/harness-saa-s-release-notes) and Self-Managed Enterprise Edition release notes are available [here](/release-notes/self-managed-enterprise-edition).
 :::
 
+
+## April 05, 2023, version 79001
```
[the remark this answers is not in the document]
Comment (charanya-j) [reply release-notes/cloud-cost-management.md:17]: CCM is deployed independently

**Review by roshnisarangadharan — APPROVED** [review]
Approved with minor suggestions

**Pull request merged**